sim/ofat.py

```diff
@@ -121,6 +126,31 @@ def analyse(root_path):
         "tp": tps
     })
 
+def plot(root_path):
+
+    dirs = glob.glob(root_path + "/*/stats.csv")
+
+    values = pd.DataFrame(index =[range(1, len(dirs)+1)])
+    branches=['branch:']
+
+    for directory in dirs:
+        path = Path(directory)
+        sample_name = path.parts[-2]
+        branches.append(sample_name)
+        csv = pd.read_csv(directory)
+        values[sample_name] = csv['f_score']
+
+    
+    plt.boxplot(values, showfliers = True)
+    
+    #Define y axis (optional)
+    y = (0.85, 0.9, 0.95, 1)
```

Comment:
Could make a default argument to the function?

Reply:
Left this as is since if you wanted to leave it blank (e.g. if you were unsure what values to use) ranges will be autogenerated so easier in this setup, since can just be commented out then

Reply:
Hmm, ok, I would be tempted to leave out. Consider someone running this on different data, the plots might not fit on the hard-coded range.

Reply:
Ok yeah thats fair, I was assuming f_scores wouldnt drop below .85 but it is a possibility